Plugins/Public/minecontrol_plugin/Main.cpp

```diff
@@ -533,30 +352,71 @@ BOOL WINAPI DllMain(HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved)
 	// If we're being loaded from the command line while FLHook is running then
 	// set_scCfgFile will not be empty so load the settings as FLHook only
 	// calls load settings on FLHook startup and .rehash.
-	if (fdwReason == DLL_PROCESS_ATTACH && set_scCfgFile.length() > 0)
-		LoadSettings();
+	if (fdwReason == DLL_PROCESS_ATTACH) {
+		if (set_scCfgFile.length() > 0) {
+			LoadSettings();
+		}
+
+		HkLoadStringDLLs();
+	}
+	if (fdwReason == DLL_PROCESS_DETACH)
+	{
+		HkUnloadStringDLLs();
```

Comment:
I've seen this in a few places. Is this function done at global level or a plugin level? If its global, this would break every plugin that uses it after manually unloading it

Reply:
Has to be done per-plugin, otherwise any attempts at calling HkGetWStringFromIDS end up in an exception.
I'd assume unloading also only affects that specific plugin, but haven't tested that since it's not something Disco really does.